samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +349,35 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	int64_t time_now;
+	char buf[SAMPLE_MSG_BUF_SIZE];
+
+	/* Get the current timestamp */
+	err = date_time_now(&time_now);
+	if (err) {
+		LOG_ERR("Failed to get timestamp for Hello World message");
+		return err;
```

Comment:
```suggestion
		time_now = -1;
```
this should take care of it... unless the backend doesn't like a negative ts value?

Reply:
I'm using this as an ID for each message, and with ts =-1 will make the code more complex. How would I take care of that case in Python? Failed it?

Reply:
As we discussed offline, this should be a random number instead.